### PR TITLE
Featureimage/cmake migration

### PR DESCRIPTION
**Pull request by eric3873**

### CODE REVIEW

**Review by aquette — APPROVED** [review]

**Review by aquette — APPROVED** [review]
passes lintscript. Not done thorough analysis though!

File: CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,92 @@
+cmake_minimum_required(VERSION 3.13)
+cmake_policy(VERSION 3.13)
+
+project(fty-metric-store
+    VERSION 1.0.0
+    DESCRIPTION "Persistance for metrics"
```
Comment (jeffdegott) [inline CMakeLists.txt:6]: persistence

File: resources/fty-metric-store-cleaner.service.in [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+EnvironmentFile=-/usr/share/fty/etc/default/fty__%n.conf
+EnvironmentFile=-/usr/share/fty/etc/default/fty__fty-metric-store.service.conf
+EnvironmentFile=-/etc/default/bios
+EnvironmentFile=-/etc/default/bios__%n.conf
+EnvironmentFile=-/etc/default/fty
+EnvironmentFile=-/etc/default/fty__%n.conf
+EnvironmentFile=-/etc/default/fty__fty-metric-store.service.conf
+Environment="prefix=/usr"
+EnvironmentFile=/etc/default/bios-db-rw
+ExecStart=@CMAKE_INSTALL_FULL_BINDIR@/@PROJECT_NAME@-cleaner @MS_CONF_FILE@
```
Comment (jeffdegott) [inline resources/fty-metric-store-cleaner.service.in:22]: missed -c argument?

File: resources/fty-metric-store.service.in [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+EnvironmentFile=-/usr/share/bios/etc/default/bios__%n.conf
+EnvironmentFile=-/usr/share/bios/etc/default/fty
+EnvironmentFile=-/usr/share/bios/etc/default/fty__%n.conf
+EnvironmentFile=-/etc/default/bios
+EnvironmentFile=-/etc/default/bios__%n.conf
+EnvironmentFile=-/etc/default/fty
+EnvironmentFile=-/etc/default/fty__%n.conf
+Environment="prefix=/usr"
+EnvironmentFile=/etc/default/bios-db-rw
+ExecStart=@CMAKE_INSTALL_FULL_BINDIR@/@PROJECT_NAME@ @MS_CONF_FILE@
```
Comment (jeffdegott) [inline resources/fty-metric-store.service.in:20]: missed -c argument?

**Comment by jeffdegott** [comment]
@eric3873 Check the service files... you missed -c (--config) arguments

File: resources/fty-metric-store-cleaner.service.in (hunk 2 above)
Comment (eric3873) [reply resources/fty-metric-store-cleaner.service.in:22]: I just replace previous config with zproject: ExecStart=@prefix@/bin/fty-metric-store-cleaner @sysconfdir@/@PACKAGE@/fty-metric-store.cfg

File: resources/fty-metric-store.service.in (hunk 3 above)
Comment (eric3873) [reply resources/fty-metric-store.service.in:20]: ExecStart=@prefix@/bin/fty-metric-store-cleaner @sysconfdir@/@PACKAGE@/fty-metric-store.cfg

File: CMakeLists.txt (hunk 1 above)
Comment (eric3873) [reply CMakeLists.txt:6]: Sorry copy of description of the project description ...

**Pull request merged**